mono/metadata/loader.c

```diff
@@ -1768,7 +1768,7 @@ mono_get_method_checked (MonoImage *image, guint32 token, MonoClass *klass, Mono
 
 	/* We do everything inside the lock to prevent creation races */
 
-	error_init (error);
+	//error_init_internal (error); // FIXME?
```

Comment:
Why not just removing it?

Reply:
Still fiddling trying to find the right set. A few weeks ago when I was working on this, it wasn't just the MONO_API ones that were right the right set. I didn't mean for this PR to be reviewed today.